docs/reference/ml/delayed-data-detection.asciidoc

```diff
@@ -0,0 +1,42 @@
+[role="xpack"]
+[[ml-delayed-data-detection]]
+=== Working with delayed data
+
+Delayed data are documents that are indexed late. That is to say, it is data
+related to a time that the {dfeed} has already processed.
+
+When you create a datafeed, you can specify a {ref}/ml-datafeed-resource.html[`query_delay`] setting.
+This setting enables the datafeed to wait for some time past real-time, which means any "late" data in this period
+is fully index before the datafeed tries to gather it. However, if the setting is set too low, the datafeed may query
+for data before it has been indexed and consequently miss that document. Conversely, if it is set too high,
+analysis drifts farther away from real-time. The balance that is struck depends upon each use case and
+the environmental factors of the cluster.
+
+==== Why worry about delayed data?
+
+This is a particularly prescient question. If there are data delayed randomly (and consequently missing from analysis),
```

Comment:
```suggestion
This is a particularly prescient question. If data is delayed randomly (and consequently missing from analysis),
```

Reply:
@lcawl `data` is plural is it not? So it should be if `data are` yes?

Reply:
```suggestion
This is a particularly prescient question. If data are delayed randomly (and consequently missing from analysis),
```

Reply:
Apparently that's one of those questions that folks like to wrestle over. We don't have it in our style guide at this point, so either is likely fine for now.